oracle/src/services/gasPrice.js

```diff
@@ -39,7 +39,7 @@ const fetchGasPrice = async (speedType, factor, bridgeContract, oracleFetchFn) =
   const contractOptions = { logger }
   const oracleOptions = { speedType, factor, limits: GAS_PRICE_BOUNDARIES, logger }
   cachedGasPrice =
-    (await gasPriceFromOracle(oracleFetchFn, oracleOptions)) ||
+    (await gasPriceFromSupplier(oracleFetchFn, oracleOptions)) ||
```

Comment:
We need to be consistent in this case. Consider to rename `oracleFetchFn`, `oracleOptions`, `foreignGasOracleOpts`, `oracleUrl` and other places where the oracle mentioned in terms of a gas price supplier.

Reply:
Done.